connectors/src/lib/sync_status.ts

```diff
@@ -25,6 +25,20 @@ async function syncFinished(
   return new Ok(undefined);
 }
 
+export async function reportInitialSyncProgress(
+  connectorId: ModelId,
+  progress: string
+) {
+  const connector = await Connector.findByPk(connectorId);
+  if (!connector) {
+    return new Err(new Error("Connector not found"));
+  }
+  connector.firstSyncProgress = progress;
```

Comment:
What's the current behavior for Notion?

Reply:
It's undefined for now. We discussed it with @fontanierh and decided that reporting the status as a string was the best bay to make sure it fits everyone's usecase).

Reply:
What will be visible in the UI? Just "synchronizing" for Notion with the current state of this PR?